src/components/AudioPlayer/Buttons/SelectReciterMenu.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import useTranslation from 'next-translate/useTranslation';
+import { useDispatch, useSelector } from 'react-redux';
+
+import CheckIcon from '../../../../public/icons/check.svg';
+import ChevronLeftIcon from '../../../../public/icons/chevron-left.svg';
+
+import DataFetcher from 'src/components/DataFetcher';
+import PopoverMenu from 'src/components/dls/PopoverMenu/PopoverMenu';
+// import { playbackRates } from 'src/components/Navbar/SettingsDrawer/AudioSection';
```

Comment:
I think we should remove the comments and un-used code unless you think we will need it in the future